alphapept/gui/start.py

```diff
@@ -65,3 +66,11 @@ def start():
             "https://datashare.biochem.mpg.de/s/p8Qu3KolzbSiCHH/download",
         )
         st.write("Put both files in one folder and go to the New experiment tab.")
+
+        
+    latest_version = get_latest_version()
+    if latest_version and VERSION_NO != latest_version:
```

Comment:
Note that this can be interpreted as `(latest_version and VERSION_NO) != latest_version`, which is syntactically correct but not the intended implementation. It is best to make this explicit by writing this line as `latest_version and (VERSION_NO != latest_version)`

Reply:
Done :)